tools/integration_tests/write_large_files/random_write_large_file_test.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2023 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package write_large_files
+
+import (
+	"crypto/rand"
+	"log"
+	rand2 "math/rand"
+	"os"
+	"path"
+	"syscall"
+	"testing"
+
+	"github.com/googlecloudplatform/gcsfuse/tools/integration_tests/util/operations"
+	"github.com/googlecloudplatform/gcsfuse/tools/integration_tests/util/setup"
+)
+
+const NumberOfRandomWriteCalls = 20
+const DirForRandomWrite = "dirForRandomWrite"
+const MinWritableByteFromFile = 0
```

Comment:
Is this variable really needed? All I see this being used for is being subtracted from file-size.